README.md

```diff
@@ -48,9 +68,25 @@ or
 pip install git+https://github.com/The-AI-Alliance/gofannon.git
 ```
 
+## Communication Channels
+- **Disccord**: Real-time collaboration and support
+- **Github Discussions**: Technical deep dives and asynchhronous collaboration
```

Comment:
update with link found here
https://github.com/The-AI-Alliance/gofannon/blob/main/website/community/index.md?plain=1#L16